test/mocks/upstream/cluster_info.cc

```diff
@@ -53,14 +53,14 @@ MockUpstreamLocalAddressSelector::MockUpstreamLocalAddressSelector(
 MockClusterInfo::MockClusterInfo()
     : http2_options_(::Envoy::Http2::Utility::initializeAndValidateOptions(
           envoy::config::core::v3::Http2ProtocolOptions())),
-      stat_names_(stats_store_.symbolTable()),
+      traffic_stats_names_(stats_store_.symbolTable()),
```

Comment:
s/traffic_stats_names_/traffic_stat_names_/ here and in similar places.

Reply:
Done